pom.xml

```diff
@@ -78,12 +78,18 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>credentials</artifactId>
+                <version>2.3.7</version>
```

Comment:
Instead of adding this, update the version for `bom-2.176.x` to 11.

Reply:
addressed in aeebf177dccd985fbed72fe0492134b6d1a3d610